src/containers/SignupPage.js

```diff
@@ -21,31 +21,39 @@ class SignupPage extends PureComponent {
 
 	render() {
 
-		const { authenticated } = this.props
-
-		if(!authenticated) return (
-			<div>
-				<Paper style={{
-					display: 'inline-block',
-					margin: 50,
-					width: 450,
-					height: 300,
-					padding: 20,
-					textAlign: 'center',
-					lineHeight: 1.6,
-				}}
-				zDepth={2}
-				>
-					<h2>Het lijkt erop dat je geen geregistreerde klant bent.</h2>
-					<p>Als je een uitnodiging van Flexicon hebt gekregen en de link je naar deze pagina brengt,
-						neem alsjeblieft contact op met de contactpersoon in de e-mail.</p>
-				</Paper>
-		</div>
-	)
-
-		if (this.props.signup.success) return (
-			<Redirect to="/flexicon/create/order" />
-		)
+	// 	const { authenticated } = this.props
+	//
+	// 	if(!authenticated) return (
```

Comment:
we don't want to show this anymore? it works if you try to get to the signup page without a valid jwt

Reply:
The point is, the user is not supposed to be logged in when entering the signup page, so this page should be accessible without any restriction. The jwt we are using in the link is not the same as the one used to log in the user. The idea is, if you do not have a valid and with the right payload jwt in the link, then the page will do nothing because the backend will throw a bad request error

Reply:
well it would also be accessible for hackers then. or if something doesn't work for some reason the user wouldn't know because he/she sees the same page. that's why I made this part. but if we all agree that we don't want it then sure, let's delete it 😅